### DISCUSSION TITLE
hide version

### DISCUSSION THREAD

**Discussion by chpego** [dani-garcia/vaultwarden#2752] [Ideas]
When i read documentation about [Hardening-Guide](https://github.com/dani-garcia/vaultwarden/wiki/Hardening-Guide), i look into shodan search, and i see a lot of warden instance with version appaear.

I haven't looked into whether or not this is possible on the Bitwarden version, but maybe it would be interesting to hide the Vaultwarden application **version** on the authentication screen?
![image](https://user-images.githubusercontent.com/38792705/190945965-c2555766-54be-4d4e-823d-641a84b22e93.png)

**Comment by Masgalor** [comment]
See https://github.com/dani-garcia/vaultwarden/discussions/2458
